fetcher/fetcher.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```

Comment:
Because the methods in the tar fetcher are tighted to tar archiving. We can see what we can split out but I propose to do it as a separate PR. Cause that is more like a refactoring commit.